src/dropin-model.js

```diff
@@ -258,6 +258,8 @@ function isPaymentOptionEnabled(paymentOption, options) {
   return SheetView.isEnabled({
     client: options.client,
     merchantConfiguration: options.merchantConfiguration
+  }).catch(function () {
+    return Promise.resolve(false);
```

Comment:
Should we `console.log` the error here along with the sheet id? Might be good to expose the exception if it gets thrown.